### ISSUE TITLE
General Polaris Test Plan > 3 - My Account > 3.2 - Verify that you can edit your account information

### ISSUE THREAD

**Issue by martypowell** [baltimorecounty/BCPL-assets#525]
Steps to reproduce:
1. Log into account.
2. Under Contact Information and Preferences, edit a field.
3. Click: Submit Change Request.
4. The expected behavior is for an acknowledgment of the change request to appear in the Messages section.
 
Notes:
In ILS-Test, after I log into my account, I expanded the “Contact Information and Preferences” section.
The “State/Province” field was already set to Maryland, but the “County” field was empty, so I entered “Baltimore County” and clicked the “Submit Change Request” button.
The page reloaded, but no message was displayed.
In the Messages section, it said, “No Messages.”
To my surprise, both the “State/Province” field and the “County” field were now empty.
I repeated the action by entering “Baltimore County” into the County field (and Maryland in the “State/Province” field) and clicking the “Submit Change Request” button.
Again, the page reloaded and no message was displayed.
Both fields were empty once again.
For a comparison, I tried the above procedure in Polaris prod and the outcome was different.
After I clicked the “Submit Change Request” button, the page reloaded and displayed a bold, red message in the Messages section, which said, “The following change was requested: Change County to: Baltimore County / Your change request has been submitted. Your account information will be changed when the library processes the request. Please bring a valid ID to your local branch to verify this change before checking out items.”
The “County” field was empty, but I wasn’t concerned because of the relevant message.

**Comment by martypowell** [comment]
@danfox01 I have a feeling maybe this never works in test, because there is some sort of work flow.

**Comment by danfox01** [comment]
@martypowell, this is definitely possible.

@jdomasky, do you know if this feature worked in test while we were using 5.5?

**Comment by jdomasky** [comment]
On May 16, after I edited my account in ILS-Test by changing the text message preference, I clicked the submit button and a red message was displayed in the Messages section acknowledging the change.
As we know, yesterday, when I tried to edit my account in ILS-Test, I entered a county into the empty County field, and nothing happened after submission. In comparison, this worked in Polaris prod.
It occurred to me these might be apples and oranges, so I tried to change my text message preference in ILS-Test just now ...and it worked! A red message appeared in the Messages section.
Based on this experience, and also based on the fact that the County can be changed in prod, I believe this feature will work after the June 24 upgrade.
I suggest we consider this resolved.
